src/twitter.js

```diff
@@ -155,7 +155,7 @@ class Twitter {
 					text = text.replace(url.url, displayUrlHtml);
 
 					if(targetUrl.startsWith("https://") && !targetUrl.startsWith("https://twitter.com/")) {
-						medias.push(`<a href="${targetUrl}"><img src="https://v1.opengraph.11ty.dev/${encodeURIComponent(targetUrl)}/small/" alt="OpenGraph image for ${displayUrl}" loading="lazy" decoding="async" width="375" height="197" class="tweet-media tweet-media-og"></a>`);
+						medias.push(`<template data-island><a href="${targetUrl}"><img src="https://v1.opengraph.11ty.dev/${encodeURIComponent(targetUrl)}/small/onerror/" alt="OpenGraph image for ${displayUrl}" loading="lazy" decoding="async" width="375" height="197" class="tweet-media tweet-media-og" onerror="this.parentNode.remove()"></a></template>`);
```

Comment:
@zachleat the `onerror/` at the end of the `src` feels strange, is it an undocumented option of the OpenGraph Image service?

Reply:
yeah, that’s new as of this morning: https://github.com/11ty/api-opengraph-image

Reply:
Some implementation notes on needing a successful Builder Functions cache (to avoid repeating these requests too often) simultaneously while wanting `<img onerror>` to fire https://github.com/11ty/api-opengraph-image/blob/83e1995e5b9c75595750068a04caf2475b89925f/functions/og/index.js#L22